roles/datastore/datastore_setup/tasks/setup_datastore.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Create the iati-datastore tables
```

Comment:
Would look to split this into two `yml` files - one to create the various tables in the database, the other to add things to cron.
(from a maintenance point of view, it isn't immediately clear that `setup_datastore.yml` is the place to go to update the crontab)

Reply:
done